### PR TITLE
Spookylukey change default autoconvert behaviour

### PR DESCRIPTION
**Pull request by benjaoming**
Supersedes #202

### CODE REVIEW

File: djmoney/models/fields.py [hunk 1]
```diff
@@ -34,12 +34,17 @@
 if 'djmoney_rates' in settings.INSTALLED_APPS:
     try:
         from djmoney_rates.utils import convert_money
```
Comment (Stranger6667) [inline djmoney/models/fields.py]: IMO it is a bad thing - to import models on module level. In some cases it leads to "app registry is not ready" errors.

File: tests/test_models.py [hunk 2]
```diff
@@ -351,7 +347,7 @@ def test_patching(self):
 
 
 rates_is_available = pytest.mark.skipif(
-    not AUTO_CONVERT_MONEY,
+    not RATES_INSTALLED,
```
Comment (Stranger6667) [inline tests/test_models.py]: Seems it is not imported

File: djmoney/models/fields.py (hunk 1 above)
Comment (benjaoming) [inline djmoney/models/fields.py]: These things should be handled on a conventional level IMO.

A convention to *never* download anything in `djmoney/__init__.py` is very reasonable! We already have an issue open on this.

I've opened up #208 to explicitly deal with this.

File: tests/test_models.py (hunk 2 above)
Comment (benjaoming) [inline tests/test_models.py]: Yeah, absolutely right.. something seems to have been renamed... added in 9c88515

**Comment by Stranger6667** [comment]
I'll take care about this PR

**Comment by benjaoming** [comment]
You might have to re-open it. I can't make Travis build the correct commit.. the failing lint test isn't reproducable locally with the same version of isort, so I suspect Travis for building the wrong commit dbceff3.

**Comment by benjaoming** [comment]
re-open = closing this and the creating a brand new PR

**Comment by benjaoming** [comment]
Merged in #210

**Pull request closed without merging**